Client/Frontend/ContextualHint/ContextualHintEligibilityUtility.swift

```diff
@@ -103,15 +104,15 @@ struct ContextualHintEligibilityUtility: ContextualHintEligibilityUtilityProtoco
             // Retrieve the last timestamp for Fakespot CFRs
             let lastTimestamp = profile.prefs.timestampForKey(PrefsKeys.FakespotLastCFRTimestamp)
             // Check if 24 hours have passed since the last timestamp
-            let hasTimePassed = lastTimestamp != nil ? Date.hasTimePassedBy(hours: 24, lastTimestamp: lastTimestamp!) : false
+            let hasTimePassed = lastTimestamp != nil ? Date.hasTimePassedBy(hours: 12, lastTimestamp: lastTimestamp!) : false
 
-            if cfrCounter == 1, !hasOptedIn, hasTimePassed {
+            if cfrCounter <= 2, !hasOptedIn, hasTimePassed {
                 // - Display CFR-1
-                profile.prefs.setInt(2, forKey: PrefsKeys.ContextualHints.shoppingOnboardingCFRsCounterKey.rawValue)
+                profile.prefs.setInt(cfrCounter+1, forKey: PrefsKeys.ContextualHints.shoppingOnboardingCFRsCounterKey.rawValue)
```

Comment:
Please add spaces before and after the `+`. Pretty sure Swiftlint will complain otherwise.